pyonfx/shape.py

```diff
@@ -877,7 +906,7 @@ def rotate_on_axis_z(point, theta):
         return shape.move()
 
     @staticmethod
-    def star(edges, inner_size, outer_size):
+    def star(edges: float, inner_size: float, outer_size: float):
```

Comment:
```suggestion
    def star(edges: int, inner_size: float, outer_size: float) -> Shape:
```
Edges is int, not float.